src/main/webapp/app/modules/canvas/filter/filter-element.tsx

```diff
@@ -208,13 +204,10 @@ const mapStateToProps = (storeState: IRootState) => ({
   featuresList: storeState.feature.entities,
   selectedFilter: storeState.filter.selectedFilters,
   filterStateChange: storeState.filter.filterStateChange,
```

Comment:
why do we have filterStateChange?

Reply:
filterStateChange  is use to reflect the top header and right side panel when filter, clear filter or bookmark is apply.